### PR TITLE
Update authz-assembly-change.md

### LINKED ISSUE #15745: AddAuthorization Suggested Actions do not Work
In the section "Authorization: AddAuthorization overload moved to different assembly" it is stated that:

    New behavior
    AddAuthorization methods exist in Microsoft.AspNetCore.Authorization.Policy. AddAuthorizationCore is the new name for the old methods.

And that the recommended action is:

    Either add a reference to Microsoft.AspNetCore.Authorization.Policy or use AddAuthorizationCore instead.

The package `Microsoft.AspNetCore.Authorization.Policy` does not contain the method `AddAuthorization` as of the latest version (2.2.0). I need my authorization library to support both .NET Core 2.2, and .NET Core 3.0 apps, so it's important to have a solution to this problem.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: df2066cb-244d-0b1b-cf46-428b198e8601
* Version Independent ID: e52631de-28de-77c2-082f-e79b89b17f04
* Content: [Breaking changes, version 2.2 to 3.0 - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/compatibility/2.2-3.0#feedback)
* Content Source: [docs/core/compatibility/2.2-3.0.md](https://github.com/dotnet/docs/blob/master/docs/core/compatibility/2.2-3.0.md)
* Product: **dotnet-core**
* GitHub Login: @gewarren
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

Adjust paragraph per PM feedbeck in issue.

Fixes #15745

### CODE REVIEW

File: includes/core-changes/aspnetcore/3.0/authz-assembly-change.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-The core `AddAuthorization` methods that used to reside in `Microsoft.AspNetCore.Authorization` were renamed to `AddAuthorizationCore`. The old `AddAuthorization` methods still exist, but are in the `Microsoft.AspNetCore.Authorization.Policy` package instead. Apps using both methods should see no impact. Apps that weren't using the policy package must switch to using `AddAuthorizationCore`.
+The core `AddAuthorization` methods that used to reside in `Microsoft.AspNetCore.Authorization` were renamed to `AddAuthorizationCore`. The old `AddAuthorization` methods still exist, but are in the `Microsoft.AspNetCore.Authorization.Policy` assembly instead. Apps using both methods should see no impact. Note that `Microsoft.AspNetCore.Authorization.Policy` now ships in the shared framework rather than a standalone package as discussed in [Shared framework: Assemblies removed from Microsoft.AspNetCore.App](#shared-framework-assemblies-removed-from-microsoftaspnetcoreapp).
 
 #### Version introduced
 
 3.0
 
 #### Old behavior
-
+note
```
Comment (tdykstra) [inline includes/core-changes/aspnetcore/3.0/authz-assembly-change.md]: What is this?

**Review by tdykstra — COMMENTED** [review]
One question.

**Comment by mairaw** [comment]
Also, don't these ASP.NET breaking changes exist somewhere else as well @scottaddie? We might need to update other places.

**Comment by scottaddie** [comment]
@mairaw The only other place the content exists is in the associated GitHub issue.

File: includes/core-changes/aspnetcore/3.0/authz-assembly-change.md (hunk 1 above)
Comment (scottaddie) [inline includes/core-changes/aspnetcore/3.0/authz-assembly-change.md]:
```suggestion

```

Comment (Thraka) [reply includes/core-changes/aspnetcore/3.0/authz-assembly-change.md]: no idea. I used the github editor.

**Review by scottaddie — APPROVED** [review]

**Pull request merged**